src/java.desktop/windows/classes/sun/awt/Win32GraphicsDevice.java

```diff
@@ -463,13 +477,9 @@ public synchronized void setDisplayMode(DisplayMode dm) {
             WWindowPeer peer = AWTAccessor.getComponentAccessor().getPeer(w);
             configDisplayMode(screen, peer, dm.getWidth(), dm.getHeight(),
                 dm.getBitDepth(), dm.getRefreshRate());
-            // resize the fullscreen window to the dimensions of the new
-            // display mode
-            Rectangle screenBounds = getDefaultConfiguration().getBounds();
-            w.setBounds(screenBounds.x, screenBounds.y,
-                        screenBounds.width, screenBounds.height);
-            // Note: no call to replaceSurfaceData is required here since
-            // replacement will be caused by an upcoming display change event
+            // Note: window will get resized to actual full-screen dimensions
+            // in the upcoming display change event, when the DPI scales
+            // would already be correctly set etc. (see JDK-8282863)
```

Comment:
We usually do not mention the bugid in the comments, just the description of how it works is fine.

Reply:
@mrserb the mentioned bugid reference is now removed